src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -294,9 +211,10 @@ public TransformationStatus Transform(ReadOnlySpan<byte> source, Span<byte> dest
         /// <returns>Number of bytes written to the buffer.</returns>
         public static int DecodeInPlace(Span<byte> buffer)
         {
+            // TODO: Fix
```

Comment:
I removed the following API which DecodeInPlace relied on:
```C#
private static bool Decode(byte b0, byte b1, byte b2, byte b3, out byte r0, out byte r1, out byte r2)
```

I essentially have to make similar changes that I made to the implementation of the Transform API to the DecodeInPlace, which I will do in subsequent PR.

Also, since I changed the decoding map, it means the implementation needs to be updated.